eng/Versions.props

```diff
@@ -233,12 +237,15 @@
     <MicrosoftWebAdministrationPackageVersion>11.1.0</MicrosoftWebAdministrationPackageVersion>
     <MicrosoftWebXdtPackageVersion>1.4.0</MicrosoftWebXdtPackageVersion>
     <SystemIdentityModelTokensJwtPackageVersion>6.7.1</SystemIdentityModelTokensJwtPackageVersion>
-    <!-- Packages from 2.1/2.2 branches used for site extension build -->
+    <!-- Packages from 2.1, 2.2, 3.1 and 5.0 branches used for site extension build. -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3-servicing-20163-14</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
```

Comment:
Should this be 3.1.7-servicing-20372-13 now❔ That shipped (sort of) with 3.1.7.

Reply:
@BrennanConroy is this your call❔ If not you, who❔

Reply:
Likely my call, I think it doesn't matter if it stays or updates (assuming the 3.1.3-servicing bits stay in the feed). I'll double check tomorrow.

Reply:
We probably should keep this as the latest. Anyway to automate that?

Reply:
Doubt this is possible to automate given oddities with circular subscriptions. But, let's chat w/ @mmitche and @wtgodbe in the 3.1.9 timeframe. Could you file an issue while I'm thinking of other problems with this PR please❔ (Placing it in https://github.com/orgs/dotnet/projects/10 will help us remember.)